### PR TITLE
4.2.0

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by core23** [comment]
Do we need a release for PHPDoc changes?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.2.0](https://github.com/sonata-project/SonataAdminBundle/compare/4.1.0...4.2.0) - 2021-10-29
+### Added
+- [[#7561](https://github.com/sonata-project/SonataAdminBundle/pull/7561)] Make AuditReaderInterface generics. ([@VincentLanglet](https://github.com/VincentLanglet))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#7561](https://github.com/sonata-project/SonataAdminBundle/pull/7561)] Make `AuditReaderInterface` generic ([@VincentLanglet](https://github.com/VincentLanglet))
```

**Comment by VincentLanglet** [comment]
> Do we need a release for PHPDoc changes?

Yes, because it fix phpstan build on DoctrineORM and you cannot use this without a release.

**Review by phansys — DISMISSED** [review]

**Comment by core23** [comment]
> Yes, because it fix phpstan build on DoctrineORM and you cannot use this without a release.

This might be true from our CI perspective. 

When looking from the user perspective, this minor release does not add any new features or fixes. Some months ago we had this discussion about PHPDoc changes and came to the conclusion that these changes are pedantic.

Just my two cents...

**Comment by VincentLanglet** [comment]
> When looking from the user perspective, this minor release does not add any new features or fixes. Some months ago we had this discussion about PHPDoc changes and came to the conclusion that these changes are pedantic.

This is not true. To me it's either a patch or a minor.

A user with static analysis will see new issues or will have some issues solved. So there are changes.

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**